modules.json

```diff
@@ -20,6 +20,11 @@
                         "git_sha": "b6d4d476aee074311c89d82a69c1921bd70c8180",
                         "installed_by": ["modules"]
                     },
+                    "ensemblvep/vep": {
```

Comment:
We changed the nf-core version of ensemblvep and moved it to the `local` folder. Hence this statement triggers automatic download and addition of the ensemblvep module, if `nf-core modules lint` is run and should be removed.